src/transformers/models/convnext/modeling_convnext.py

```diff
@@ -209,8 +209,9 @@ class ConvNextEncoder(nn.Module):
     def __init__(self, config):
         super().__init__()
         self.stages = nn.ModuleList()
-        drop_path_rates = [x.item() for x in torch.linspace(0, config.drop_path_rate, sum(config.depths))]
-        cur = 0
```

Comment:
I've removed this (unclear) cur variable, and instead implement it similar to Ross' implementation in timm.